### DISCUSSION TITLE
Column headers and gene symbol in downloads

### DISCUSSION THREAD

**Discussion by oleraj** [geneontology/helpdesk#457] [Ideas]
Hi,

It would be nice to add column headers for the downloads, including the links to download -- e.g., [this link](https://golr-aux.geneontology.io/solr/select?defType=edismax&qt=standard&indent=on&wt=csv&rows=100000&start=0&fl=source,bioentity_internal_id,bioentity_label,qualifier,annotation_class,reference,evidence_type,evidence_with,aspect,bioentity_name,synonym,type,taxon,date,assigned_by,annotation_extension_class,bioentity_isoform&facet=true&facet.mincount=1&facet.sort=count&json.nl=arrarr&facet.limit=25&csv.encapsulator=&csv.separator=%09&csv.header=false&csv.mv.separator=%7C&fq=document_category:%22annotation%22&fq=isa_partof_closure:%22GO:0002376%22&facet.field=aspect&facet.field=taxon_subset_closure_label&facet.field=type&facet.field=evidence_subset_closure_label&facet.field=regulates_closure_label&facet.field=isa_partof_closure_label&facet.field=annotation_class_label&facet.field=qualifier&facet.field=annotation_extension_class_closure_label&facet.field=assigned_by&facet.field=panther_family_label&q=*:*) from a gene ontology page, but also downloads using the Download button.
<img width="126" alt="image" src="https://github.com/geneontology/helpdesk/assets/10860409/388ca496-018e-44c8-8ac4-40ea268a49ee">
Without column headers it's difficult to determine what the column contents are since these are non-standard format.

I also find it inconvenient that there is no gene symbol column available for downloads. This is a column in the GAF file so I believe it is in the database but for some reason is not available in the downloads.  This is one of the most useful annotations to have for downstream processes so I would want it to be included by default. Any reason it is not available?

Thanks!

Andrew

**Comment by kltm** [comment]
@oleraj 

What is the link for the page you're referring to? I believe you're likely looking at the "Annotations" tab of a term page (e.g. https://amigo.geneontology.org/amigo/term/GO:2000469)?

If so, I believe what you are looking for is in the "Available pool", under the label `Gene/product (bioentity_label)`. It turns out that this has been noticed before and I am bumping it: https://github.com/geneontology/amigo/issues/471. In the interim, you can move the item from "Available pool" over.

Unfortunately, given restrictions on how the current stack is built, we are unable to offer column headers at this time. Because of this, all that we can offer at this point is this "pseudo-GAF" for downloads. This is a known issue and we have several tickets open for it:

https://github.com/geneontology/amigo/issues/94
https://github.com/geneontology/amigo/issues/194
https://github.com/geneontology/amigo/issues/414

This should be addressed in a future AmiGO release.
